Kickstarter-iOS/Views/Cells/DashboardActionCell.swift

```diff
@@ -74,8 +74,8 @@ internal final class DashboardActionCell: UITableViewCell, ValueCell {
 
     self.viewModel.outputs.goToMessages
       .observeForUI()
-      .observeValues { [weak self] project in
-        self?.delegate?.goToMessages(self, project: project)
+      .observeValues { [weak self] _ in
```

Comment:
Should the output stop emitting a project here or is the output reused elsewhere?